iroh-net/src/tls/certificate.rs

```diff
@@ -26,6 +28,45 @@ const P2P_SIGNING_PREFIX: [u8; 21] = *b"libp2p-tls-handshake:";
 // Similarly, hash functions with an output length less than 256 bits MUST NOT be used.
 static P2P_SIGNATURE_ALGORITHM: &rcgen::SignatureAlgorithm = &rcgen::PKCS_ECDSA_P256_SHA256;
 
+#[derive(Debug)]
+pub(crate) struct AlwaysResolvesCert(Arc<rustls::sign::CertifiedKey>);
```

Comment:
This stuff is copied from https://github.com/libp2p/rust-libp2p/pull/5498 (for anyone wondering about this in code review)